test/read-pacts.test.ts

```diff
@@ -28,4 +29,37 @@ describe('readPacts', () => {
             'Pact directory ./pacts is empty.'
         );
     });
+
+    test('throws error when pact interactions array is empty', () => {
+        mockFs({
+            './pacts': {
+                'service-pact.json': JSON.stringify(
+                    pactMockFactory.build({
+                        provider: {
+                            name: 'provider-service',
+                        },
+                        interactions: [],
+                    })
+                ),
+                'another-service-pact.json': JSON.stringify(
+                    pactMockFactory.build({
+                        provider: {
+                            name: 'another-provider-service',
+                        },
+                        interactions: [],
+                    })
+                ),
+            },
+        });
+
+        expect(() => readPacts('./pacts')).toThrow(
+            `Pact interactions for provider: another-provider-service in pact file ${path.join(
```

Comment:
maybe the better option would be moving ${path.join(...)} to const above expect() ?

And what do you think about using `.toThrowErrorMatchingInlineSnapshot` ? https://jestjs.io/docs/expect#tothrowerrormatchinginlinesnapshotinlinesnapshot

Reply:
Moved to const for readability - and regarding inline snapshot - there is a case of operating system path separator - different for linux and windows, that's why path.join is used here.